index.org

```diff
@@ -317,24 +318,23 @@ See [[https://github.com/rtic-scope/examples/blob/master/src/bin/blinky-noconf.r
 
```

Comment:
This applies to the points above: What does consume mean in this case? And effectively how? Is it up to the programmer to create a comparator that can be consumed effectively? It is ambiguous.

Reply:
The DWT comparator is a hardware resource that the user must assign during `init`. See https://github.com/rtic-scope/examples/blob/master/src/bin/blinky.rs#L49 for example.